### ISSUE TITLE
Accordion header error not announced to assistive technology

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/vets-design-system-documentation#5059]
### Point of contact

Jamie Klenetsky Fay / ADE

### Defect Severity

1, Launchblocker. Critical. Must be fixed before launch.

### Severity rationale

Content that's changed dynamically isn't announced to assistive technology.

### URL where issue occurs

https://staging.va.gov/supporting-forms-for-claims/accrued-benefits-application-21p-601

### Component or pattern (if applicable)

va-accordion-header

### Environment details

- OS: Windows 11
- Screen reader: JAWS
- Browser: Edge

### Steps to reproduce

1. In Step 1, eligibility, answer "no" to the two questions
2. Fill out the form until you get to the Review page (Step 8)
3. Turn on JAWS
4. Open the first accordion (Step 1, eligibility)
5. Change one of the radio buttons from "no" to "yes"

### Observed behavior

The heading's content changes, but that change isn't announced to JAWS. I'm forced to shift-tab up to the accordion heading to hear the change.

### Expected behavior / Acceptance criteria

The new content is announced (via aria-live, presumably) to screen readers.

### Supporting evidence (screenshots, recordings, code snippets)

https://github.com/user-attachments/assets/55a72146-8c27-4661-9449-5f8839b85eaa

### Type of issue

- [ ] Axe-core
- [x] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else
